### PR TITLE
add: `joplin-desktop-app`

### PR DESCRIPTION
**Pull request by lfromanini**

### CODE REVIEW

**Comment by lfromanini** [comment]
This PR solves issue [#4983](https://github.com/pacstall/pacstall-programs/issues/4983).

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+repology=("project: joplin-desktop")
+arch=("amd64")
+pkgver="2.13.15"
+url="https://github.com/laurent22/joplin/releases/download/v${pkgver}/Joplin-${pkgver}.AppImage"
+hash="e6d2ce3400a165a889aaf2bf960d4d1931f72d80657b32009ad42f9e0d2734d7"
+makedepends=("libfuse2")
+depends=("libfuse2")
+
+package() {
+  sudo chmod +x "Joplin-${pkgver}.AppImage"
```
Comment (vigress8) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:15]:
```suggestion
```
The below line takes care of making it executable.

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  sudo install -Dm755 "Joplin-${pkgver}.AppImage" "${pkgdir}/usr/bin/Joplin.AppImage"
```
Comment (vigress8) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:16]:
```suggestion
  sudo install -Dm755 "Joplin-${pkgver}.AppImage" "${pkgdir}/usr/bin/${gives}"
```

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  wget -q -O "joplin-desktop.png" "https://joplinapp.org/images/Icon512.png"
+  sudo install -Dm644 "joplin-desktop.png" "${pkgdir}/usr/share/pixmaps/joplin-desktop.png"
+  sudo mkdir -p "${pkgdir}/usr/share/applications/"
+  echo "[Desktop Entry]
+    Encoding=UTF-8
+    Name=Joplin
+    Comment=Joplin for Desktop
+    Exec=Joplin.AppImage %u
```
Comment (vigress8) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:24]:
```suggestion
    Exec=joplin-desktop %u
```

**Review by vigress8 — CHANGES_REQUESTED** [review]

**Comment by lfromanini** [comment]
Done, @vigress8 .

**Review by vigress8 — APPROVED** [review]

**Comment by lfromanini** [comment]
@vigress8 , sorry, can you approve again after merging branch with master? Also, please, feel free to merge.

**Comment by vigress8** [comment]
@Elsie19 ?

**Review by vigress8 — APPROVED** [review]

**Comment by Elsie19** [comment]
Wait.

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript [hunk 4]
```diff
@@ -0,0 +1,31 @@
+name="joplin-desktop-app"
+gives="joplin-desktop"
+homepage="https://joplinapp.org/"
```
Comment (Elsie19) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:3]: Single quotes.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript [hunk 5]
```diff
@@ -0,0 +1,31 @@
+name="joplin-desktop-app"
+gives="joplin-desktop"
+homepage='https://joplinapp.org/'
+pkgdesc="Joplin - the secure note taking and to-do app with synchronisation capabilities for Windows, macOS, Linux, Android and iOS."
```
Comment (Elsie19) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:4]:
```suggestion
pkgdesc="Joplin - the secure note taking and to-do app with synchronisation capabilities for Windows, macOS, Linux, Android and iOS"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript [hunk 6]
```diff
@@ -0,0 +1,31 @@
+name="joplin-desktop-app"
+gives="joplin-desktop"
+homepage='https://joplinapp.org/'
+pkgdesc="Joplin - the secure note taking and to-do app with synchronisation capabilities for Windows, macOS, Linux, Android and iOS"
+maintainer="lfromanini <lfromanini@yahoo.com>"
+repology=("project: joplin-desktop")
+arch=("amd64")
+pkgver="2.13.15"
```
Comment (Elsie19) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:8]: pkgver should be higher up.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript (hunk 6 above)
Comment (lfromanini) [reply packages/joplin-desktop-app/joplin-desktop-app.pacscript:8]: No, 2.13.15 is the stable one. The higher ones are pre-release.

Comment (Elsie19) [reply packages/joplin-desktop-app/joplin-desktop-app.pacscript:8]: I meant the line should physically be higher up 😆

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**